src/main/java/com/bean00/RequestProcessor.java

```diff
@@ -1,17 +1,18 @@
 package com.bean00;
 
 import java.io.IOException;
-import java.io.Writer;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
-public class MessageController {
+public class RequestProcessor {
 
     public Response processRequest(Request request) throws IOException {
+        String requestMethod = request.getRequestMethod();
         String requestURL = request.getRequestURL();
         Path path = getFullPath(requestURL);
 
+        boolean isHEADRequest = requestMethod.equals("HEAD");
```

Comment:
I'm okay with this but some may think it's overkill to create variables out of something like this because it's pretty clear what `requestMethod.equals("HEAD")` means

Reply:
I've removed `isHEADRequest` and `rootDirectoryWasRequested`